org.mwc.debrief.core/src/org/mwc/debrief/core/editors/PlotEditor.java

```diff
@@ -1612,8 +1660,16 @@ protected void filesDropped(final String[] fileNames)
     // ok, iterate through the files
     for (int i = 0; i < fileNames.length; i++)
     {
+
```

Comment:
I'd like to suggest one change here, please @ayeshare 

After this loop, Debrief updates the plot and resizes it to show all data.  This will give the effect that, if the user has zoomed in an area, then drags in a video, then the plot will zoom out to show the full plot.

I more user-friendly logic would be something like:
* if there's a single file, and it's a video file - send it to the video player
else
* run through the rest of this method, as normal: loop through files, load them, resize plot.

Reply:
Sure, I will make that change. Thanks